.github/ISSUE_TEMPLATE/devtools_translate.md

```diff
@@ -0,0 +1,16 @@
+---
+name: DevTools translate
+about: Translate a DevTools blog post
+title: '[devtools-translate] Language - What's new in DevTools (Chrome XX)'
+labels:
+- 'devtools-translate'
+assignees: 'chybie'
+
+---
+
+Post: https://developer.chrome.com/blog/new-in-devtools-xx/
```

Comment:
Can you please add a note to the top of this file that users can delete? We get enough spam as it is. Something like [what's here](https://github.com/GoogleChrome/developer.chrome.com/blob/main/.github/ISSUE_TEMPLATE/feature_request.md).

> ⚠️ This is for feature requests with the https://developer.chrome.com site, not Chromium itself. If you want to file a bug with Chromium (the open-source project behind Google Chrome and other browsers), please use https://crbug.com.